packages/components/src/notice/style.scss

```diff
@@ -60,6 +60,7 @@
 }
 
 .components-notice-list {
-	min-width: 300px;
+	// The notice should never be wider than the viewport, or the close button might be hidden. Especially relevant at high zoom levels.
```

Comment:
@tellthemachines Super-minor, but would you mind adding a direct URL to the core ticket here? That'll give folks something to refer back to and check if this can be re-assessed later on. Once that's all set, we're ready to go!

Reply:
Done!